src/miscwrappers.cpp

```diff
@@ -522,40 +528,48 @@ syscall(long sys_num, ...)
 # endif // ifdef __x86_64__
 #endif // ifndef DISABLE_SYS_V_IPC
 # ifndef __aarch64__
+# ifndef __riscv
   case SYS_poll:
   {
     SYSCALL_GET_ARGS_3(struct pollfd *, fds, nfds_t, nfds, int, timeout);
     ret = poll(fds, nfds, timeout);
     break;
   }
+# endif
 # endif // ifdef __aarch64__
 # ifndef __aarch64__
+# ifndef __riscv
   case SYS_epoll_create:
   {
     SYSCALL_GET_ARG(int, size);
     ret = epoll_create(size);
     break;
   }
+# endif
 # endif // ifdef __aarch64__
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 13) && __GLIBC_PREREQ(2, 4)
 # ifndef __aarch64__
+# ifndef __riscv
```

Comment:
Same comment as above.  When I do 'man inotify_init', the man page also shows me 'inotify_init1'